packages/protocol/test/governance/vesting.ts

```diff
@@ -1014,8 +967,8 @@ contract('Vesting', (accounts: string[]) => {
   describe('#pause()', () => {
     it('revoker should be able to pause the vesting', async () => {
       await createNewVestingInstanceTx(vestingDefaultSchedule, web3)
-      const vestingInstanceRegistryAddress = await vestingFactoryInstance.vestings(beneficiary)
-      const vestingInstance = await VestingInstance.at(vestingInstanceRegistryAddress)
+      const vestingInstanceAddress = await vestingFactoryInstance.vestings(beneficiary)
```

Comment:
Lots of duplicated code, pull into a `beforeEach`?

Reply:
true, but there are tests in-between which change the behaviour of hte vesting scheme and construct a new vesting with that, hence I cannot set them the 3 lines in a common beforeEach